lib/spack/docs/conf.py

```diff
@@ -353,3 +369,11 @@ class SpackStyle(DefaultStyle):
 
 # How to display URL addresses: 'footnote', 'no', or 'inline'.
 #texinfo_show_urls = 'footnote'
+
+
+# -- Extension configuration -------------------------------------------------
+
+# sphinx.ext.intersphinx
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/3", None),
```

Comment:
https://docs.python.org/object.inv just redirects to https://docs.python.org/3/object.inv